WorkloadManagementSystem/DB/JobDB.sql

```diff
@@ -46,7 +46,7 @@ CREATE TABLE `Jobs` (
   `MasterJobID` INT(11) UNSIGNED NOT NULL DEFAULT 0,
   `Site` VARCHAR(100) NOT NULL DEFAULT 'ANY',
   `JobName` VARCHAR(128) NOT NULL DEFAULT 'Unknown',
-  `Owner` VARCHAR(32) NOT NULL DEFAULT 'Unknown',
+  `Owner` VARCHAR(64) NOT NULL DEFAULT 'Unknown',
```

Comment:
there should be an entry in the wiki of DIRAC then to update operationally the length